### DISCUSSION TITLE
All Pages Wrappers Return Undefined for all pages expect main content

### DISCUSSION THREAD

**Discussion by RakulAgn** [GrapesJS/grapesjs#6150] [General]
```
const pages = editor.Pages.getAllWrappers();
pages.map((page, index) => {
    const getAllLinks = page?.view;
    console.log(index, getAllLinks);
});      
```

This is Going to Return all the page views but it return only the first page view how can i get all the page views ?

@artf

**Comment by artf** [comment]
why do you need views of not rendered components?

**Reply by RakulAgn** [reply]
I want to add Google Fonts to all pages when the editor loads. My current approach is to loop through all pages to find a specific wrapper type. However, I'm encountering an issue where only the rendered page is providing the wrapper, while the remaining pages are undefined. Is there an alternative solution to achieve this goal?

@artf

**Reply by artf** [reply]
why not to add it on the model level instead of the view?
